plugins/destination/duckdb/client/write.go

```diff
@@ -154,3 +170,74 @@ func (c *Client) WriteTableBatch(ctx context.Context, sc *arrow.Schema, records
 
 	return nil
 }
+
+func transformSchema(sc *arrow.Schema) *arrow.Schema {
+	fields := sc.Fields()
+	for i := range fields {
+		fields[i].Type = transformType(fields[i].Type)
+	}
+	md := sc.Metadata()
+	return arrow.NewSchema(fields, &md)
+}
+
+func transformType(dt arrow.DataType) arrow.DataType {
+	switch {
+	case arrow.TypeEqual(dt, types.ExtensionTypes.UUID) ||
+		arrow.TypeEqual(dt, types.ExtensionTypes.Inet) ||
+		arrow.TypeEqual(dt, types.ExtensionTypes.MAC) ||
+		arrow.TypeEqual(dt, types.ExtensionTypes.JSON) ||
+		dt.ID() == arrow.STRUCT:
+		return arrow.BinaryTypes.String
+	case arrow.IsListLike(dt.ID()):
+		return arrow.ListOf(transformType(dt.(*arrow.ListType).Elem()))
+	default:
+		return dt
+	}
+}
+
+func transformRecord(sc *arrow.Schema, rec arrow.Record) arrow.Record {
+	cols := make([]arrow.Array, rec.NumCols())
+	for i := 0; i < int(rec.NumCols()); i++ {
+		cols[i] = transformArray(rec.Column(i))
+	}
+	return array.NewRecord(sc, cols, rec.NumRows())
+}
+
+func transformArray(arr arrow.Array) arrow.Array {
+	dt := arr.DataType()
+	switch {
+	case arrow.TypeEqual(dt, types.ExtensionTypes.UUID) ||
+		arrow.TypeEqual(dt, types.ExtensionTypes.Inet) ||
+		arrow.TypeEqual(dt, types.ExtensionTypes.MAC) ||
+		arrow.TypeEqual(dt, types.ExtensionTypes.JSON) ||
+		dt.ID() == arrow.STRUCT:
```

Comment:
```suggestion
	case arrow.TypeEqual(dt, types.ExtensionTypes.UUID),
		arrow.TypeEqual(dt, types.ExtensionTypes.Inet),
		arrow.TypeEqual(dt, types.ExtensionTypes.MAC),
		arrow.TypeEqual(dt, types.ExtensionTypes.JSON),
		dt.ID() == arrow.STRUCT:
```

Reply:
What's the different here? why it should be that one or the otherway? either way I think it's either in lint or not.